sw/device/tests/README.md

```diff
@@ -14,13 +14,18 @@ For FPGA / silicon this may be a host machine, while for UVM / Verilator this ma
 
 ## Self Checking Mechanism
 
-Currently, the self-checking mechanism is accomplished through a console message.
-When the tests pass, it will output `PASS!\r\n`.
+The self-checking mechanism is accomplished by invoking one of the test status indication APIs defined in `sw/device/lib/testing/test_status.h`.
```

Comment:
Why is this called a "self-checking mechanism"? As a non-DV person reading this, this comes off as a integrity self-test rather than the actual assertion mechanism.